### DISCUSSION TITLE
Bardic Inspiration

### DISCUSSION THREAD

**Discussion by Ramz1337** [DFreds/dfreds-convenient-effects#122] [Q&A] [answered]
Hi,

Not sure if right area but new to the whole VTT area and installed your brilliant module and setup Bardic Inspiration which works 99% however when the Actor that has the effect makes a attack (e.g. with a axe) the Attack roll box comes up  and to continue on is to use the Bardic Inspiration or click close. The question is have I done something wrong in setting it up etc. 

![image](https://user-images.githubusercontent.com/98008544/151959430-b00e78f7-fed7-4d80-bade-71c086226f0b.png)

Many thanks in advance if you can point me in the right direction.

**Comment by DFreds** [comment, marked as answer]
Yeah, it's an optional usage you can do after you see the roll. If you use it, the extra die will be added and the effect will be removed. Hitting close is choosing not to use it for that roll, and it will not be removed.

**Reply by Ramz1337** [reply]
Many thanks I think I was focused on looking for a 2nd button to pass over the BI choose but pressing close does the job lol :)
